content/en/docs/setup/independent/self-hosting.md

```diff
@@ -0,0 +1,61 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Configuring your kubernetes cluster to self-host the control plane
+content_template: templates/concept
+weight: 100
+---
+
+{{% capture overview %}}
+
+### Self-hosting the Kubernetes control plane {#self-hosting}
+
+As of 1.8, you can experimentally create a _self-hosted_ Kubernetes control
+plane. This means that key components such as the API server, controller
+manager, and scheduler run as [DaemonSet pods](/docs/concepts/workloads/controllers/daemonset/)
+configured via the Kubernetes API instead of [static pods](/docs/tasks/administer-cluster/static-pod/)
+configured in the kubelet via static files.
+
+To create a self-hosted cluster see the `kubeadm alpha selfhosting pivot` command.
+
+#### Caveats
+
+1. This feature pivots your cluster into a unsupported state, rending kubeadm unable
+  to manage you cluster anylonger. This includes `kubeadm upagrade`.
```

Comment:
this is mostly copy and pasted, but I updated this to try and make it clear that it was not supported.

Reply:
please move this sentence in a caution block like so, above Caveats:
```
{{< caution >}}
This feature pivots your cluster into a unsupported state, rendering kubeadm unable
to manage you cluster any longer. This includes `kubeadm upagrade`.
{{< /caution >}}
```
fixed a couple of typos, too.

Reply:
@neolit123 sorry I missed this previously. I have updated it. Thank you.